src/components/ui/formik/validationSchemas.ts

```diff
@@ -98,9 +99,9 @@ export const vipValidationSchema = (
 
 export const ipBlockValidationSchema = Yup.string()
   .required('A value is required.')
-  .matches(IP_ADDRESS_BLOCK_REGEX, {
+  .matches(SUBNET_CIDR_REGEX, {
     message:
-      'Value "${value}" is not valid IP block address, expected value is IP/netmask. Example: 123.123.123.0/24', // eslint-disable-line no-template-curly-in-string
+      'Value "${value}" is not valid IP block address, expected value is IP/netmask. Address mask size must be between 1 to 25 and must include at least 128 addresses. Example: 123.123.123.0/24', // eslint-disable-line no-template-curly-in-string
```

Comment:
```suggestion
      'Value "${value}" is not a valid IP address block. Expected value is IP/netmask. The netmask must be between 1-25 and include at least 128 addresses. Example: 123.123.123.0/24', // eslint-disable-line no-template-curly-in-string
```

Slight tweaks to consider. Aligns on the word `netmask` for clarity.

Reply:
Updated + enhanced with tweaks suggested by Ori.